packages/vscode-common/getExtensionApi.ts

```diff
@@ -1,41 +1,42 @@
 import type {
```

Comment:
This file is worth a look, as this was where most of the violations were coming from